actpool/queueworker.go

```diff
@@ -0,0 +1,269 @@
+package actpool
+
+import (
+	"context"
+	"encoding/hex"
+	"math/big"
+	"strings"
+	"sync"
+	"sync/atomic"
+
+	"github.com/iotexproject/go-pkgs/cache/ttl"
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"go.uber.org/zap"
+
+	"github.com/iotexproject/iotex-core/action"
+	accountutil "github.com/iotexproject/iotex-core/action/protocol/account/util"
+	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/iotexproject/iotex-core/pkg/tracer"
+)
+
+type (
+	workerJob struct {
+		ctx context.Context
```

Comment:
to make it simple, we can choose not to have `ctx` here, context can relate to worker, see comment below

Reply:
the ctx is to make job cancelable

Reply:
can pass ctx to Start(), see below comment